DOCS/client-api-changes.rst

```diff
@@ -32,6 +32,10 @@ API changes
 
 ::
 
+ --- mpv 0.39.0 ---
+ 2.4    - mpv_render_param with the MPV_RENDER_PARAM_ICC_PROFILE argument no
+          longer has incorrect assumptions about memory allocation and can be
+          safely used.
```

Comment:
there's basically no way any api user can use it before this
```suggestion
          correctly used.
```

Reply:
changed